qa/tasks/cephfs/mount.py

```diff
@@ -170,7 +169,8 @@ def assert_and_log_minimum_mount_details(self):
                               sudo=True).decode())
 
     def is_mounted(self):
-        return self.mounted
+        return self.hostfs_mntpt in \
+            self.client_remote.read_file('/proc/self/mounts',stdout=StringIO())
```

Comment:
What happens when it cant find that path or file? Does it return `None` or throw some error?

Reply:
It will.

Reply:
So there can be no corner cases?

Reply:
It will throw some error, is what I mean.